src/main/java/org/myrobotlab/framework/Service.java

```diff
@@ -2814,4 +2817,8 @@ protected void registerForInterfaceChange(Class<?> clazz) {
     Runtime.getInstance().registerForInterfaceChange(getClass().getCanonicalName(), clazz);
   }
 
+  public Map<String, ServiceConfig> getDefault() {
```

Comment:
please correct the return type here. it should be   LinkedHashMap<String, ServiceConfig>   otherwise services could create an unordered configset.

Reply:
this could also be "final" method so that no subclass could override it.